ci/gpu/build.sh

```diff
@@ -13,6 +13,7 @@ function logger() {
 # Set path and build parallel level
 export PATH=/conda/bin:/usr/local/cuda/bin:$PATH
 export PARALLEL_LEVEL=4
+export CUDA_REL=${CUDA_VERSION%.*}
```

Comment:
Why `CUDA_VERSION`? gpuCI uses `CUDA`.

Reply:
gpuCI uses `CUDA_VERSION` from what I see when the environment is printed. Just tried using `CUDA` and its undefined.